bazel/envoy_build_system.bzl

```diff
@@ -187,6 +187,10 @@ def envoy_cc_library(
         deps = deps + [envoy_external_dep_path(dep) for dep in external_deps] + [
             repository + "//include/envoy/common:base_includes",
             repository + "//source/common/common:fmt_lib",
+            envoy_external_dep_path("abseil_flat_hash_map"),
+            envoy_external_dep_path("abseil_flat_hash_set"),
+            envoy_external_dep_path("abseil_node_hash_map"),
```

Comment:
Are we using node_hash_map/set right now? If not can we remove until they are actually used? Same below.